Package.swift

```diff
@@ -0,0 +1,133 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "AutomatticTracksiOS",
+    platforms: [.macOS(.v10_14), .iOS(.v12)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "AutomatticTracks",
+            targets: ["AutomatticTracksEvents",
+                      "AutomatticRemoteLogging",
+                      "AutomatticExperiments",
+                      "AutomatticCrashLoggingUI",
+                      "AutomatticTracksModel",
+                      "AutomatticTracksModelObjC",
+                      "AutomatticTracks",
+            ]),
+
+        // Xcode 12 has an issue where the first build after
+        // cleaning fails if there is a dependency that vends
+        // a binary .xcframework. We are working around this
+        // on CI by first building this "_WorkaroundSPM"
+        // target that _only_ builds that one dependency.
+        // We ignore any failures when building this target.
+        // Then we go on to build the actual product, which
+        // builds correctly.
```

Comment:
Yes, it works fine on Xcode 13. When I originally put up this PR, Xcode 13 was still a few months out so I was shooting for compatibility with Xcode 12. But since the final Xcode 13 build is likely to be coming out this week, we may be able to just drop this altogether. (On the other hand, this workaround doesn't cause any problems in Xcode 13 either.)

Reply:
Excellent! Thank you for the update 🙇 

In my experience in Automattic Apps Division so far, we haven't been jumping onto the latest Xcode version as soon as released because of the various projects and CI setups. So, I'd lean towards shipping with this and drop it later on.